Core/Core/Logging/SpeckleLog.cs

```diff
@@ -236,11 +241,27 @@ SpeckleLogConfiguration logConfiguration
     }
 
     var logger = serilogLogConfiguration.CreateLogger();
+
+    // Configure scope after logger created.
+    SentrySdk.ConfigureScope(scope =>
+    {
+      scope.User = new User { Id = id };
+    });
+    SentrySdk.ConfigureScope(scope =>
+    {
+      scope.SetTag("hostApplication", hostApplicationName);
+    });
+
     if (logConfiguration.LogToFile && !canLogToFile)
     {
       logger.Warning("Log to file is enabled, but cannot write to {LogFilePath}", logFilePath);
     }
 
+    if (s_isMachineIdUsed)
+    {
+      logger.Warning("Cannot set user id for the global log context.");
```

Comment:
Would we want to add more than just a warning log here?

I can see adding a ´.Enrich.WithProperty("isMachineId", true)` would allow us to separate logs that came from users with valid computed IDs versus the ones that come from users with machine ID.

Not sure if this would bring much benefit though. @BovineOx WDYT?

Reply:
it is make sense and added.